CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/utilities/MultiValueStore.java

```diff
@@ -31,55 +31,75 @@ public class MultiValueStore implements ParameterReadAccess, ParameterWriteAcces
     private float f;
     private double d;
     private Object o;
+    
+    public int getInt(){
+        return i;
+    }
+
+    public long getLong() {
+        return l;
+    }
+
+    public float getFloat() {
+        return f;
+    }
+
+    public double getDouble() {
+        return d;
+    }
+    
+    public Object getObject(){
```

Comment:
should these be made private (I'm guessing they were only meant to be used by this class)?